### DISCUSSION TITLE
Beta申し込み時の組織名の半角スペース

### DISCUSSION THREAD

**Discussion by netmarkjp** [topotal/waroom-roadmap#12] [Others]
入力した文字列がうろ覚えなので不確かですが、Beta申し込み時に入力した組織名の半角スペースが消えているかも？

`株式会社X-Tech 5` （5の前に半角スペース）が正式名称ですが、Web画面で入力して通ったのは `株式会社X-Tech5` （半角スペースなし）でした。

わたしが申し込みしたときに半角スペースをいれてないかもなので勘違いだったらスミマセン。

**Comment by sawa-zen** [comment]
@netmarkjp 
確認したところ現状仕様として組織名に半角スペースは許容していませんでした。おそらくですが入力した際にエラートーストが出て半角を消して試していただいたんだと思います。（もし間違っていたらすみません 🙇🏻 ）

組織名に使用可能な文字列がなんなのか現状の画面では分かりづらいのでこちら修正しようと思います。ありがとうございます 👍🏻

**Comment by netmarkjp** [comment]
> 確認したところ現状仕様として組織名に半角スペースは許容していませんでした

なるほど。ならおそらくわたしが自分で消してますね。失礼しました 😇
